<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQ3</class>
 <widget class="QDialog" name="CQ3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CQ3</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>851</width>
     <height>611</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QListWidget" name="m_pListWidget">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>颜色设置</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>路径位置</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="title">
     <string/>
    </property>
    <widget class="QStackedWidget" name="m_pStackWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>541</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="m_pLbPath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>当前路径：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="m_pLePath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>12</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="m_pBtnPath">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>11</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择路径</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="m_pLbColor">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>当前颜色：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_pBtnColor">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择颜色</string>
       </property>
      </widget>
      <widget class="QLabel" name="m_pLbColorPalatte">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
